package.json

```diff
@@ -12,9 +12,9 @@
     "build:vm": "yarn workspace near-mock-vm run build:wasm",
     "test": "yarn workspaces run test && yarn check:all",
     "doc": "rm -rf docs; typedoc --tsconfig tsconfig.json && touch docs/.nojekyll",
-    "prettify": "prettier --write '**/src/**/*.ts'",
+    "prettify": "prettier --write **/src/**/*.ts",
     "lint": "eslint --max-warnings 0 --ext ts \"*/assembly/**/*.ts\"",
-    "check:pretty": "yarn prettier --check '**/src/**/*.ts'",
+    "check:pretty": "yarn prettify --check **/src/**/*.ts",
```

Comment:
Does this mean it will check and write in the same step?

Reply:
I don't know, I just fix these commands. At least, it cannot run normally in my windows.

Reply:
And there is no `yarn prettier` cmd.

Reply:
Oh I see.  It should just be prettier.  Does `prettier --check **/src/**/*.ts` work?

On unix, `yarn prettier` is the same as `prettier`, that is it will just execute the binary of the some name.  But from within a npm script you shouldn't need to use it and can call the binary directly.

Reply:
it works.